components/employees/add-employee-form/index.tsx

```diff
@@ -0,0 +1,453 @@
+                    <SelectTrigger
+                      className={`${field?.value === undefined ? "text-placeholder" : "text-textColor"}`}
+                    >
+                      <SelectValue placeholder={holidayCalender?.placeholder} />
+                    </SelectTrigger>
+                  </FormControl>
+                  <SelectContent>
+                    {/* TODO: Holiday Calender will be added later */}
+                    {/* <SelectItem></SelectItem> */}
+                    <div>Not Found</div>
+                  </SelectContent>
+                </Select>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name="gender"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>{gender?.label}</FormLabel>
+
+                <Select
+                  onValueChange={field.onChange}
+                  defaultValue={field?.value}
+                >
+                  <FormControl>
+                    <SelectTrigger
+                      className={`${field?.value === undefined ? "text-placeholder" : "text-textColor"}`}
+                    >
+                      <SelectValue placeholder={gender?.placeholder} />
+                    </SelectTrigger>
+                  </FormControl>
+                  <SelectContent>
+                    {GENDERS?.map((gender, index) => (
+                      <SelectItem key={index} value={gender?.value}>
+                        {gender?.label}
+                      </SelectItem>
+                    ))}
+                  </SelectContent>
+                </Select>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name={"probationEnd"}
+            render={({ field }) => {
+              return (
+                <FormItem>
+                  <FormLabel>{probationEnd?.label}</FormLabel>
+
+                  <FormControl>
+                    <DatePicker
+                      onSelect={field.onChange}
+                      placeholder={probationEnd?.placeholder}
+                      value={field?.value}
+                    />
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              );
+            }}
+          />
+
+          <FormField
+            control={form.control}
+            name="directManagerId"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>{directManager?.label}</FormLabel>
+
+                <Select
+                  onValueChange={field.onChange}
+                  defaultValue={field?.value}
+                >
+                  <FormControl>
+                    <SelectTrigger
+                      className={`${field?.value === undefined ? "text-placeholder" : "text-textColor"}`}
```

Comment:
here as well

Reply:
@imrankhan-stixor , this is not an input field so `placeholder:text-placeholder` will not work